src/main/java/repositories/UserDataBase.java

```diff
@@ -0,0 +1,38 @@
+package repositories;
+
+import models.User;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class UserDataBase {
```

Comment:
1) formatting issue: it should be a blank line between methods
2) it would be great to add a contract for the current class

Reply:
- [x] added a blank line between methods
- [x] created UserDataBase interface
- [x]  renamed UserDataBase class to DataBase and it implements UserDataBase interface